pyproject.toml

```diff
@@ -70,12 +70,35 @@ include = [
   "/.coveragerc",
 ]
 
-[tool.black]
-line-length=80
-extend-exclude="_vendor"
+# Ruff section
+[tool.ruff]
+lint.select = [
+    "I",      # isort: all
+    "PL",     # pylint: all
+    "S",      # flake8-bandit: all
+    "N",      # pep8-naming: all
+    "RUF100"  # ruff: find unused noqa
+]
+exclude = ["_vendor"]
+
+# Same as Black.
+line-length = 80
+indent-width = 4
+
+[tool.ruff.format]
+# Like Black, use double quotes for strings.
+quote-style = "double"
+
+# Like Black, indent with spaces, rather than tabs.
+indent-style = "space"
+
+# Like Black, respect magic trailing commas.
+skip-magic-trailing-comma = false
+
+# Like Black, automatically detect the appropriate line ending.
+line-ending = "auto"
```

Comment:
do we need all of this configuration?  If not, I would rather not have it clutter pyproject.toml

Reply:
I will test it, and remove if not necessary